spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/batch/SimpleJobService.java

```diff
@@ -256,14 +257,19 @@ public int countJobs() {
 	@Override
 	public int stopAll() {
 		Collection<JobExecution> result = jobExecutionDao.getRunningJobExecutions();
-		Collection<String> jsrJobNames = getJsrJobNames();
+		Collection<String> jsrJobNames = getJobNames();
 
 		for (JobExecution jobExecution : result) {
-			if (jsrJobOperator != null && jsrJobNames.contains(jobExecution.getJobInstance().getJobName())) {
-				jsrJobOperator.stop(jobExecution.getId());
-			} else {
-				jobExecution.stop();
-				jobRepository.update(jobExecution);
+			if (jsrJobNames.contains(jobExecution.getJobInstance().getJobName())) {
```

Comment:
**[NIT]** I think if we invert this we can short circuit and throw the stop exception rather than have to indent the try/catch deeper.